cli/scripts/publish_docker_scraper_images.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh
+tag=$(git describe --abbrev=0 --tags)
+read -r -p "Do you wish to also deploy $tag?" yn
+case $yn in
+    [Yy]* ) ./docsearch docker:build local_tag && docker push algolia/docsearch-scraper-base:"$tag" && docker push docsearch-scraper:"$tag";;
+    [Nn]* ) ./docsearch docker:build;;
+    * ) echo "Please answer yes or no, aborting."; exit 1;;
```

Comment:
FAR (= For Another Review): decide a default behavior (so that we can just hit enter)

Reply:
I want to make it explicit and avoid any wrong push. Let's use this concern for another iteration.